site/config-overrides.md

```diff
@@ -205,61 +205,61 @@ The following `jdbc-testDS.xml` override template demonstrates setting the URL,
 ```
 
 ---
-# Step-by-Step Guide
+# Step-by-step guide
 
-* Create a directory containing (A) a set of situational configuration templates for overriding the mbean properties you want to replace and (B) a version.txt file.
+* Create a directory containing (A) a set of situational configuration templates for overriding the MBean properties you want to replace and (B) a `version.txt` file.
   * This directory must not contain any other files.
-  * The version.txt file must contain only the string `2.0`.
-  * Templates must not override the settings listed in [Unsupported Overrides](#unsupported-overrides).
-  * Templates must be formatted and named as per [Override Template Names and Syntax](#override-template-names-and-syntax) and [References](#references).
-  * Templates can embed macros that reference environement variables or Kubernetes secrets.  See [Override Template Macros](#override-template-macros).
-* Create a kubernetes config map from the directory of templates.
-  * The config map must be in the same kubernetes namespace as the domain.
-  * If the config map is going to be used by a single DOMAIN_UID, it is recommended to add the 'weblogic.domainUID=<mydomainuid>' label to help track the resource.
-  * For example, assuming './mydir' contains your version.txt and situation config template files:
+  * The `version.txt` file must contain only the string `2.0`.
+  * Templates must not override the settings listed in [Unsupported overrides](#unsupported-overrides).
+  * Templates must be formatted and named as per [Override template names and syntax](#override-template-names-and-syntax) and [References](#references).
+  * Templates can embed macros that reference environement variables or Kubernetes secrets.  See [Override template macros](#override-template-macros).
+* Create a Kubernetes configuration map from the directory of templates.
+  * The configuration map must be in the same Kubernetes namespace as the domain.
+  * If the configuration map is going to be used by a single `DOMAIN_UID`, it is recommended to add the `weblogic.domainUID=<mydomainuid>` label to help track the resource.
+  * For example, assuming `./mydir` contains your `version.txt` and situation configuration template files:
     ```
     kubectl -n MYNAMESPACE create cm MYCMNAME --from-file ./mydir
     kubectl -n MYNAMESPACE label cm MYCMNAME weblogic.domainUID=DOMAIN_UID
```

Comment:
kubectl -n MYNAMESPACE label cm MYCMNAME weblogic.domainUID=DOMAIN_UID
 should be 
"kubectl -n MYNAMESPACE label cm MYCMNAME weblogic.domainUID=<DOMAIN_UID>"

Reply:
The existing DOMAIN_UID without the <> is better.    The convention throughout this document is to purely use caps to represent replaceable values - as you can likely see from the other replaceable values in the same command line.